src/hitchhiker/tree/utils/cljs/async.cljc

```diff
@@ -1,11 +1,8 @@
-(ns hitchhiker.tree.utils.async
-  (:require
-   [hitchhiker.tree.utils.platform :as platform]
-   #?(:clj [clojure.core.async :as async]
-      :cljs [cljs.core.async :as async :include-macros true])))
+(ns hitchhiker.tree.utils.cljs.async
```

Comment:
Should this be a .cljs file?

Reply:
No, this is required because macros compile in Clojure. 
So this file has two passes. Once in Clojure and again for ClojureScript. The ClojureScript one requiring the macros created from the Clojure pass. 
https://github.com/replikativ/hitchhiker-tree/blob/development/src/hitchhiker/tree/utils/cljs/async.cljc#L2